ComputeDataDrivenCrossSection.py

```diff
@@ -27,13 +27,28 @@
 parser.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
 parser.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
 parser.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+parser.add_argument("--Lc", action='store_true', help='flag to compute Lc cross section', default=False)
 parser.add_argument("--batch", action='store_true', help='suppress video output', default=False)
+parser.add_argument("--corr", action='store_true', help='assume raw yield and (prompt/feed-down) fraction uncertanties as correlated', default=False)
```

Comment:
Since we use it mainly for the non-prompt measurements, I would suggest to change the default to `corr` as true (you could check the `add_mutually_exclusive_group` here https://docs.python.org/3/library/argparse.html, which might be useful in this case)